core/templates/base-components/oppia-footer.component.ts

```diff
@@ -53,6 +63,40 @@ export class OppiaFooterComponent {
       return 'https://medium.com/oppia-org';
     }
   }
+
+  validateEmailAddress(): boolean {
+    let regex = new RegExp(AppConstants.EMAIL_REGEX);
+    return regex.test(String(this.emailAddress));
+  }
+
+  subscribeToMailingList(): void {
+    this.mailingListBackendApiService.subscribeUserToMailingList(
+      String(this.emailAddress),
+      // Most of the times, since the name is not provided
```

Comment:
Most of the time, the name is `null` since it is not provided.

Reply:
Thanks, Done